koku/api/resource_types/aws_category/serializers.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Serializer for aws category resource types."""
+import logging
+
+from rest_framework import serializers
+
+LOG = logging.getLogger(__name__)
```

Comment:
We should not import logging and create this logger if we are not going to use it. If you use this locally for debugging, that is fine if that is how you work, but don't leave it here. It is unused code.

Reply:
This was honestly just a sloppy copy & paste error, so I went back to where I copied it from and removed it from there. Which made me wonder where else we import logging and don't use it. Which lead me down a rabbithole, but I think I got all instances where we import it and don't use it.